README.md

```diff
@@ -2,6 +2,70 @@
 
 A project that presents an iRODS Zone as HTTP v1.1.
 
+## Quickstart (Running from Docker Hub Image)
+
+1. Generate a local configuration JSON file.
```

Comment:
I don't think the numbering is needed since each step is followed by a code block and the quickstart section is very short. This also makes the instructions similar to the rest of the document.

The indentation that markdown applies also makes it less appealing visibly. Perhaps there's a way to tell markdown not to indent the numbering.

Reply:
agreed.  removing numbering.